inc/command/class-command.php

```diff
@@ -26,7 +26,8 @@ protected function configure() {
 		$this->setDefinition( [
 			new InputArgument( 'subcommand', InputArgument::REQUIRED, 'phpunit | codecept' ),
 			new InputOption( 'chassis', null, null, 'Run commands in the Local Chassis environment' ),
-			new InputOption( 'module', 'm', InputArgument::OPTIONAL, 'Run commands for a specific module', 'project' ),
+			new InputOption( 'path', 'p', InputArgument::OPTIONAL, 'Use a custom path for tests folder.', '../tests' ),
+			new InputOption( 'output', 'o', InputArgument::OPTIONAL, 'Use a custom path for output folder.', '' ),
```

Comment:
Sorry I was referring to codeception's own `-o / --override` option here rather than adding one 😬 I think this likely easier to use for this use-case at least, and the codeception option can still be passed using `-o` or `--override` after the options delimiter but it might be a bit confusing.

I think the equivalent would be something like `composer dev-tools codecept run -- -o 'settings: paths: output: ../custom/output/path'`

Reply:
Yes, I think the more we simplify it the better. We can explain this in the docs to eliminate the confusion.